modules/ROOT/pages/apikit-4-implement-rest-api.adoc

```diff
@@ -0,0 +1,257 @@
+                    "address":
+                      {
+                        "street": "Douglas Extension",
+                        "suite": "Suite 847",
+                        "city": "McKenziehaven",
+                        "zipcode": "59590-4157",
+                        "geo": { "lat": "-68.6102", "lng": "-47.0653" },
+                      },
+                    "phone": "1-463-123-4447",
+                    "website": "ramiro.info",
+                    "company":
+                      {
+                        "name": "Romaguera-Jacobson",
+                        "catchPhrase": "Face to face bifurcated interface",
+                        "bs": "e-enable strategic applications",
+                      }
+                  }
+                ]
+--
+. Save it as `api.raml`.
+. In Studio, select *File* > *New* > *Mule Project*.
+. In *Project Name*, type the name of your Mule project.
+* If you are importing your API specification from Exchange, select *Import a published API*, click the plus icon, and select *From Exchange*.
++
+image::import-from-exchange.png["*Import a published API* highlighted when clicking on plus to select the From Exchange option"]
+
+* If you are importing your API specification from a local file in your file system, select *Import RAML from Local File*.
++
+image::import-from-local-file.png["*Import RAML from local file* highlighted in the API implementation section"]
+* If you are importing your API specification from Design Center, select *Download RAML from Design Center*.
++
+image::import-from-design-center.png["*Import from Design Center* highlighted in the API implementation section"]
+* Select *Scaffold flows from these API specifications* if not already selected.
+. Click *Finish*.
++
+APIkit generates the necessary flows based on the API specification you imported.
+. Run your APIkit project by clicking *Run* > *Run As* > *Mule Application*.
+
+== Simulate API Calls
+
+. In Anypoint Studio, locate the *APIkit Consoles* view and click *Open console* under the base point URL `+http://localhost:8081/console/+`.
++
+image::apikit-consoles.png["APIkit console tab displays the location of the basepoint URL in the window."]
++
+. Your default browser opens. The *API Console* shows the API summary and information about the project, such as the title, version, and supported protocols.
++
+image::apikit-console1.png["API console version info"]
++
+. In *API Summary*, select `GET` for the `/users` resource.
++
+It opens a detail page of the resource showing, among others, the expected JSON example for an HTTP 200 response.
++
+image::apikit-console3.png["APIkit response"]
++
+. In *API Summary*, expand `/userbyid`. 
+. Select `GET` for the `/usersbyid` resource.
++
+. Click `Try it` to expand the view for sending a request.
+. The *API Console* displays a section to add parameters and headers to try a request. Select *Show optional parameters* if not selected.
```

Comment:
```suggestion
+ The *API Console* displays a section to add parameters and headers to try a request.
. Select *Show optional parameters* if not selected.
```
The first sentence is info, the second is a step.

Reply:
Great catch! Thanks!